core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/common/AbstractUserStoreManager.java

```diff
@@ -3592,6 +3592,7 @@ protected void addInitialAdminData(boolean addAdmin, boolean initialSetup) throw
         }
         String adminUserName = UserCoreUtil.removeDomainFromName(realmConfig.getAdminUserName());
         String adminRoleName = UserCoreUtil.removeDomainFromName(realmConfig.getAdminRoleName());
+        String adminPassword = UserCoreUtil.removeDomainFromName(realmConfig.getAdminPassword());
```

Comment:
Don't change the password as mentioned in [1]

[1] https://github.com/wso2/carbon-kernel/pull/517/files#diff-ce9fb4f89114a5caaeda8d45be79d46eR3672